tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -1,66 +1,48 @@
 import logging
 
+import pytest
+
 from ocs_ci.framework.testlib import ocs_upgrade
-from ocs_ci.framework import config
-from ocs_ci.ocs.cluster import CephCluster, CephHealthMonitor
-from ocs_ci.ocs.resources.storage_cluster import (
-    ocs_install_verification
-)
-from ocs_ci.utility.utils import TimeoutSampler
-from ocs_ci.ocs.ocs_upgrade import (
-    verify_image_versions,
-    OCSUpgrade,
-)
-from ocs_ci.ocs.exceptions import TimeoutException
+from ocs_ci.ocs.disruptive_operations import worker_node_shutdown
+from ocs_ci.ocs.ocs_upgrade import run_ocs_upgrade
+
 
 log = logging.getLogger(__name__)
 
 
+@pytest.mark.parametrize(
+    argnames=["operation"], argvalues=[
+        pytest.param(
+            worker_node_shutdown, marks=pytest.mark.polarion_id("OCS-1579")
+        ),
+    ]
+)
+def test_worker_node_abrupt_shutdown(operation):
+    log.info("Starting disruptive test function: "
+             "test_worker_node_abrupt_shutdown")
+    abrupt = True
+    run_ocs_upgrade(operation, abrupt)
+
+
+@pytest.mark.parametrize(
```

Comment:
I think you can omit parametrize here and above.

Just use directly marker for polarion:
@pytest.mark.polarion_id("OCS-1575")

the operation can be directly passed in run_ocs_upgrade call like I will comment below.

Reply:
done